hadoop-hdds/docs/content/SparkOzoneFSK8S.md

```diff
@@ -78,11 +78,13 @@ And create a custom `core-site.xml`:
 <configuration>
     <property>
         <name>fs.o3fs.impl</name>
-        <value>org.apache.hadoop.fs.ozone.OzoneFileSystem</value>
+        <value>org.apache.hadoop.fs.ozone.BasicOzoneFileSystem</value>
     </property>
 </configuration>
 ```
 
+_Note_: You may also use `org.apache.hadoop.fs.ozone.OzoneFileSystem` without the `Basic` prefix. The `Basic` version doesn't support FS statistics and security tokens but can work together with older hadoop versions.
```

Comment:
This is not accurate. If I understand correctly, the BasicOzoneFileSystem also support delegation token APIs but not FS statistics.

Reply:
Ok, thanks. It's just because I have limited knowledge about the KeyProviderTokenIssuer. Can I write:

```
The `Basic` version doesn't support FS statistics and encryption zones but....
```